woocommerce/woocommerce-android
# Refresh current theme when navigating back to theme picker

Closes: #10398 


### Description

This PR adds two changes: 
- Refresh the current theme header after switching to a new theme
- Remove the currently active theme from the list of  carousel options

### Testing instructions
1. Navigate to theme picker from settings
2. Switch to a different theme and check how the header is updated
3. Verify that the list of themes available doesn't display the currently active theme.

### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/2663464/b01c4ad7-e502-4648-885e-5a6bf5c014e2
